scripts/sct_deepseg_sc.py

```diff
@@ -108,7 +108,7 @@ def crop_image_around_centerline(im_in, ctr_in, im_out, crop_size, x_dim_half, y
 
     x_lst, y_lst = [], []
     for zz in range(im_in.dim[2]):
-        x_ctr, y_ctr = center_of_mass(im_ctr.data[:, :, zz])
+        x_ctr, y_ctr = center_of_mass(data_ctr[:, :, zz])
```

Comment:
I'd put the `np.array()` cast right in there, so it is more obvious that we're doing it for `center_of_mass()`.

Reply:
True, I am modifying it